spark/v3.1/spark-runtime/pom.xml

```diff
@@ -76,6 +76,7 @@
                                     <include>org.apache.iceberg:iceberg-data</include>
                                     <include>org.apache.iceberg:iceberg-orc</include>
                                     <include>org.apache.iceberg:iceberg-parquet</include>
+                                    <include>org.apache.iceberg:iceberg-aws</include>
```

Comment:
Should we add this to `flink-runtime` module too?

Reply:
Not, the flink modules using  `org.apache.iceberg:* ` as filter